src/PKSim.Infrastructure/Serialization/ORM/Mapping/ObserverSetMetaDataMapping.cs

```diff
@@ -0,0 +1,14 @@
+﻿using FluentNHibernate.Mapping;
+using PKSim.Infrastructure.Serialization.ORM.MetaData;
+
+namespace PKSim.Infrastructure.Serialization.ORM.Mapping
+{
+   public class ObserverSetMetaDataMapping : SubclassMap<ObserverSetMetaData>
+   {
+      public ObserverSetMetaDataMapping()
+      {
+         Table("OBSERVER_SETS");
```

Comment:
New table in project database: That means that everytime a project <7.5 is opened, it will be converted....

Reply:
Will make @UlrichSi happy ;)